src/cloudai/workloads/nemo_run/cloudai_nemorun.py

```diff
@@ -613,7 +828,35 @@ def cloudai_llama3_405b_recipe() -> run.Partial:
             model_name="llama3",
         )
     )
+
+    set_enable_cuda_graphs_params(recipe)
     recipe.trainer.strategy.cross_entropy_fusion_impl = "te"
+
+    enable_fsdp = os.getenv("CLOUDAI_ENABLE_FSDP", "0") == "1"
```

Comment:
Do we have a list of all such env variables? Should we track it? Not a request for this PR.